inst/www/shared/shiny.js

```diff
@@ -3477,7 +3466,6 @@ $(document).on('click', '.slider-animate-button', function(evt) {
           val.to = Math.min(slider.result.max, slider.result.to + slider.options.step);
 
         slider.update(val);
-        forceIonSliderUpdate(slider);
```

Comment:
Well, right now Shiny won't run without the concatenated and/or minified JS, so we can't test changes locally without also running grunt locally. So we'd have to address that at the same time.